modules/ROOT/pages/migration/index.adoc

```diff
@@ -16,997 +16,198 @@ npm update @neo4j/graphql
 }
-WITH 'hello' AS this
-RETURN this
-----
 
-Which is invalid in Neo4j 5.x.
-To fix it, ensure the `RETURN` elements are aliased:
-
-[source, graphql, indent=0]
-----
-type query {
-    test: String! @cypher(statement: "RETURN 'hello' as result")
+type Person {
+    name: String
 }
 ----
-
-Another way to use this update is through an experimental option with the `columnName` flag in the `@cypher` directive:
-
+a|
 [source, graphql, indent=0]
 ----
-type query {
-    test: String! @cypher(statement: "RETURN 'hello' as result", columnName: "result")
+type ActedIn @relationshipProperties {
+    screenTime: Int
 }
-----
-
-Note that escaping strings are no longer needed in Neo4j GraphQL 4.0.0.
 
-=== `@fulltext`
-
-In version 4.0.0, a number of improvements have been made to full-text queries.
-These include the ability to return the full-text score, filter by the score and sorting by the score.
-However, these improvements required a number of breaking changes.
-
-==== Full-text queries
-
-Full-text queries now need to be performed using a top-level query, instead of being performed using an argument on a node query.
-
-As a result, the following query is now invalid:
-
-[source, graphql, indent=0]
-----
-query {
-  movies(fulltext: { movieTitleIndex: { phrase: "Some Title" } }) {
-    title
-  }
+type Movie {
+    title: String
+    actors: [Person!]! @relationship(type: "ACTED_IN", direction: IN, properties: "ActedIn")
 }
-----
-
-The new top-level queries can be used to return the full-text score, which indicates the confidence of a match, as well as the nodes that have been matched.
-They now accept the following arguments:
-
-* `phrase`: specifies the string to search for in the full-text index.
-* `where`: accepts a min/max score as well as the normal filters available on a node.
-* `sort: used to sort using the score and node attributes.
-* `limit`: used to limit the number of results to the given integer.
-* `offset`: used to offset by the given number of results.
 
-This means that, for the following type definition:
-
-[source, graphql, indent=0]
-----
-type Movie @fulltext(indexes: [{ indexName: "MovieTitle", fields: ["title"] }]) { # Note that indexName is the new name for the name argument. More about this below.
-  title: String!
+type Person {
+    name: String
 }
 ----
+|===
 
-The following top-level query and type definitions would be generated by the library:
+=== Interface directives
 
-[source, graphql, indent=0]
-----
-type Query {
-  movieFulltextMovieTitle(phrase: String!, where: MovieFulltextWhere, sort: [MovieFulltextSort!], limit: Int, offset: Int): [MovieFulltextResult!]!
-}
+To better match the default behavior of GraphQL, library directives used in interfaces will not cascade to the implementing types anymore. This means that most directives are no longer valid in interfaces and have to be defined in the implementing types. This also applies to custom directives.
```

Comment:
I understand that this is more general, but is it possible to add a code example just to clarify that for users?

```suggestion
To better match the default behavior of GraphQL, library directives used in interfaces will not cascade to the implementing types anymore. 
This means that most directives are no longer valid in interfaces and have to be defined in the implementing types. 
This also applies to custom directives.
```